pvacseq/lib/call_iedb.py

```diff
@@ -49,6 +49,8 @@ def main(args_input = sys.argv[1:]):
 
     if args.iedb_executable_path is not None:
         response = run(prediction_class_object.iedb_executable_params(args), stdout=PIPE)
+        if response.returncode == 2:
```

Comment:
What's special about return code 2?  Should it also fail on other non-zero exit codes--especially if the process was killed by a signal?  (Could we use `run`'s `check` option?)